SimTKmath/include/simmath/Optimizer.h

```diff
@@ -309,27 +309,76 @@ class SimTK_SIMMATH_EXPORT OptimizerSystem {
  *      - 2: all files are written to the current directory.
  *      - 3: output to console, and all files are written to the current
  *
+ *
+ * Encoding of Variables
+ *
+ * Inappropriate initialization of the algorithm may lead to resampling of the
+ * parameter distribution. Since some parameters may be defined in a bounded
+ * region [a, b]. That is if a user defines the standard deviation and the
+ * initial value of the parameter in a manner, where the resultant distribution
+ * chooses values out of the bounds. To overcome this the problem the user can
+ * formulate the problem as follows:
```

Comment:
This section will be useful to users, but this paragraph is unclear to me. Consider the following rewording:

> The CMA algorithm involves sampling a random distribution for the variables (parameters) in the optimization problem. If the sampled values do not lie within the variables' bounds, CMA must resample the distribution until the variables lie within the bounds. This resampling is undesirable, and may prevent the algorithm from functioning properly. There are two ways to avoid excessive resampling: